common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java

```diff
@@ -263,23 +267,23 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field(URL_FIELD, url);
         }
         if (hashValue != null) {
-            builder.field(HASH_VALUE_FIELD, hashValue);
+            builder.field(MLModel.MODEL_CONTENT_HASH_VALUE_FIELD, hashValue);
         }
         if (modelFormat != null) {
-            builder.field(MODEL_FORMAT_FIELD, modelFormat);
+            builder.field(MLModel.MODEL_FORMAT_FIELD, modelFormat);
```

Comment:
Why use MLModel fields rather than the filed in this class?

Reply:
Reuse the same variable?

Reply:
Check if the variable value is the same with this class

Reply:
If reuse, we can remove the duplicate variable. Let's keep one PR for one thing?

Reply:
I just replaced where variable value is the same with this class.

Reply:
>If reuse, we can remove the duplicate variable. Let's keep one PR for one thing? Refactoring is not highest priority for now.

Sure, reverted the changes.